### PR TITLE
Restructure and add note to "Create a VM Network"

### PR DESCRIPTION
**Pull request by jillian-maroket**
I added a note to address user confusion surrounding required information on the "Route" tab (Networks > VM Networks).

I also restructured the procedural information to match what I wrote for the PoC Guide. Moving forward, procedural steps (especially those involving settings) will be structured in this manner for better UX.

### CODE REVIEW

**Review by jeff-radick-suse — APPROVED** [review]
Looks fine

**Comment by jillian-maroket** [comment]
@jeff-radick-suse I applied the content changes to the 1.1 and 1.2 docs as well.
@LucasSaintarbor Please review. Thanks!

File: docs/networking/harvester-network.md [hunk 1]
```diff
@@ -39,23 +39,40 @@ Since the management network is built-in and doesn't require extra operations, y
...
+    - Auto(DHCP): The Harvester network controller retrieves the CIDR and gateway addresses from the DHCP server. You can specify the DHCP server address. 
 
     ![](/img/v1.2/networking/create-network-auto.png)
 
-    - Manual: You need to specify the CIDR and gateway values manually.
+    - Manual: Specify the CIDR and gateway addresses. 
 
     ![](/img/v1.2/networking/create-network-manual.png)
 
+    :::important
```
Comment (LucasSaintarbor) [inline docs/networking/harvester-network.md:72]:
```suggestion
    :::info important
```

File: versioned_docs/version-v1.1/networking/harvester-network.md [hunk 2]
```diff
@@ -39,22 +39,39 @@ Since the management network is built-in and doesn't require extra operations, y
...
+
+    - Auto(DHCP): The Harvester network controller retrieves the CIDR and gateway addresses from the DHCP server. You can specify the DHCP server address. 
+
+    ![](/img/v1.2/networking/create-network-auto.png)
+
+    - Manual: Specify the CIDR and gateway addresses. 
+
+    ![](/img/v1.2/networking/create-network-manual.png)
+
+    :::important
```
Comment (LucasSaintarbor) [inline versioned_docs/version-v1.1/networking/harvester-network.md:72]:
```suggestion
    :::info important
```

File: versioned_docs/version-v1.2/networking/harvester-network.md [hunk 3]
```diff
@@ -39,23 +39,40 @@ Since the management network is built-in and doesn't require extra operations, y
...
+    - Auto(DHCP): The Harvester network controller retrieves the CIDR and gateway addresses from the DHCP server. You can specify the DHCP server address. 
 
     ![](/img/v1.2/networking/create-network-auto.png)
 
-    - Manual: You need to specify the CIDR and gateway values manually.
+    - Manual: Specify the CIDR and gateway addresses. 
 
     ![](/img/v1.2/networking/create-network-manual.png)
 
+    :::important
```
Comment (LucasSaintarbor) [inline versioned_docs/version-v1.2/networking/harvester-network.md:72]:
```suggestion
    :::info important
```

**Review by LucasSaintarbor — COMMENTED** [review]
Small change to add "Important" label to note as shown here: 

Screenshot 2023-11-29 at 11 30 35 AM

**Review by LucasSaintarbor — APPROVED** [review]

**Pull request merged**